app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb

```diff
@@ -18,8 +18,16 @@ def self.raw_create_volume(ext_management_system, options = {})
       :size    => options["size"],
       :count   => options["count"]
     )
-    ext_management_system.autosde_client.VolumeApi.volumes_post(vol_to_create)
-    EmsRefresh.queue_refresh(ext_management_system)
+
+    new_volume = ext_management_system.autosde_client.VolumeApi.volumes_post(vol_to_create)
```

Comment:
Hey @llivne I am not able to confirm this, but the autosde_openapi_client gem docs show `volumes_post` returns a `VolumeResponse` same as `volumes_get` so I should be able to get the uuid of the new volume, can you check this when you get a chance?

Reply:
that is correct, this is an example for a POST response:
```{
  "compliant": true,
  "component_state": "CREATED",
  "historical_service": 16,
  "name": "liran-idea",
  "service": "941f1e8c-4bf0-4d26-8e19-31942c24a451",
  "size": 1,
  "status": null,
  "storage_resource": "f85d4e2b-ab05-47b8-a568-fadac48d42cc",
  "unmapped_since": null,
  "uuid": "6a17f7e4-eff3-4d12-b58b-5e156e19067a"
}```

Reply:
:+1: perfect thanks